### ISSUE TITLE
M1 app deployment issues.

### ISSUE THREAD

**Issue by Brandhsu** [vespa-engine/pyvespa#231]
Hi I am having some issues deploying the app locally on an m1 mac. Was wondering if this is a known issue?

Versions:
* python: 3.9.7
* pyvespa: 0.14
* docker: 4.3.2
* macOS 12.0.1

This is what I see in docker desktop:

<img width="272" alt="image" src="https://user-images.githubusercontent.com/43972676/147197646-c8498be6-c364-4f04-9f48-3d7ce8fd12bc.png">

When I try to create a app deployment, the application will hang as follows.

```python
from vespa.gallery import QuestionAnswering
from vespa.deployment import VespaDocker

app_package = QuestionAnswering()
vespa_docker = VespaDocker(port=8089)
app = vespa_docker.deploy(application_package=app_package)
```

```shell
Output

Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
Waiting for configuration server.
...
```
Thanks for all the work in creating a python wrapper for vespa!

**Comment by kkraune** [comment]
Thanks for reporting! this looks similar to https://github.com/vespa-engine/vespa/issues/20516 - we have not yet been able to work on this, pending an M1 resource to test on

**Comment by aressem** [comment]
@Brandhsu Vespa for M1 preview is now available. For information see the blogpost in [Preview of Vespa on ARM64](https://blog.vespa.ai/preview-of-vespa-on-arm64/).

Please note that it is not currently possible to select the image in `pyvespa` when using the `VespaDocker` class. I will make a feature request for this, but for testing it is possible to tag the `ghcr.io/vespa-engine/vespa-arm64-preview` image with `vespaengine/vespa` to trick `pyvespa` into using the native image:
```
$ docker pull ghcr.io/vespa-engine/vespa-arm64-preview:latest  
$ docker tag ghcr.io/vespa-engine/vespa-arm64-preview:latest vespaengine/vespa:latest
``` 
Then you can try to rerun your tests.

**Comment by kkraune** [comment]
https://github.com/vespa-engine/pyvespa/pull/292 makes it possible to specify image - pending pyvespa release before closing

**Comment by kkraune** [comment]
https://github.com/vespa-engine/pyvespa/issues/245 is closed, please try again with pyvespa-0.16.0 using the image in the blogpost (above), refer to https://pyvespa.readthedocs.io/en/latest/reference-api.html#vespadocker, see container_image.

Reopen if you find any issues.
